README.md

```diff
@@ -177,6 +177,181 @@ ApolloConverterFactory apolloConverterFactory = new ApolloConverterFactory.Build
         .build();
 ```
 
+## Support For Cached Responses
+
+Apollo GraphQL client allows you to cache responses, making it suitable for use even while offline. The client can be configured with 3 levels of caching:
+
+ - **HTTP Response Cache**: For caching raw http responses. These responses are stored in files.
+ - **Normalized Disk Cache**: Per node caching of responses in SQL. Persists normalized responses on disk so that they can used across sessions. 
+ - **Normalized InMemory Cache**: Optimized Guava memory cache for when cached normalized responses need not be persisted across sessions.  
```

Comment:
Optimized Guava memory cache for in memory caching as long as the App/Process is still alive.

I don't think the sessions thing is necessary to add here as it would be confusing if someone uses both the in memory and disk cache.